org.kde.skanlite.json

```
@@ -14,6 +14,8 @@
     "finish-args": [
         "--device=all",
         "--share=ipc",
+        "--share=network",
+        "--socket=system-bus",
```

```suggestion
        "--system-talk-name=org.freedesktop.Avahi",
```

This should work without disabling the system bus sandbox